### DISCUSSION TITLE
Docusaurus v3.6.0 Persistent Caching

### DISCUSSION THREAD

**Discussion by trevor-moore-dev** [facebook/docusaurus#10644] [v1-v2 migration]
Since Rspack doesn't support persistent caching yet, is there currently any reason to persist the node_modules/.cache directory between builds on our CI server when using docusaurus v3.6.0?

**Comment by slorber** [comment]
With Rspack, there's no reason no. Now not everyone will adopt Rspack immediately.

Some projects might even prefer to use Rspack in dev and Webpack in CI if persistent caching makes it faster.

But Rspack persistent caching is coming in `1.2` theoretically, so having the `.cache` folder persisted make you ready to adopt it once it lands. We'll probably reuse the same `.cache` folder to not disrupt people that already configured their CI for webpack.

**Reply by trevor-moore-dev** [reply]
Sounds good, thanks for the info, that's definitely useful. I'll be keeping my eyes and ears open for the release of `1.2` 😎
